### ISSUE TITLE
T-shirt order

### ISSUE THREAD

**Issue by sofer** [foundersandcoders/london-community-outreach#6]
I have the details

**Comment by iteles** [comment]
Do we want to use our actual logo this time?

**Comment by sohilpandya** [comment]
Yes please @iteles! 😊

I'm also thinking of doing all the colours! 😳

**Comment by sohilpandya** [comment]
Apologies for the terrible first draft. Looking for help/feedback as to how we can further improve these! ( Ilike design 1/2) 

![v-1](https://cloud.githubusercontent.com/assets/2305591/23668271/7c1162d0-0358-11e7-926d-73c50013885d.jpg)

**Comment by des-des** [comment]
Replace the word London with the word Nazareth. I think it will look better

**Comment by Cleop** [comment]
The bottom one is my least favourite. 

You could always put a 'F&C' in small text underneath the logo for the ones that have a lightbulb on the breast... if it didn't look too busy. I think those ones look nice it's just that it may be less obvious if you're wearing a jacket where you actually come from (if you're at a meetup or something).

**Comment by jsms90** [comment]
@Cleop we probably need to decide on FAC vs F&C first (finally!) https://github.com/foundersandcoders/london-programme/issues/122

**Comment by ghost** [comment]
@sohilpandya it looks like the version of the logo used on the black ones is the one with the white box background -if you look closely it looks weird - isn't there a newer version of the logo?
This one maybe? http://www.foundersandcoders.com/assets/fac-logo.png

Also the text - there used to be an official founders and coders font/text - it had a green tinge to it
Like this https://pbs.twimg.com/profile_images/534123785817829376/UE8T_TQF_400x400.png
I think the font above is different

Also the text is quite big

Not sure if it needs 'London' as the F&C brand is universal

NB these are just my personal opinions - other people may disagree and there might be official brand guidelines somewhere that neither of us know about

**Comment by sohilpandya** [comment]
Thanks for all the great feedback. Here is the latest version.

Does anyone know what the official font for Founders & Coders is? @markwilliamfirth, @Cleop made the changes you've suggested with the exception of the font. 

![v-3](https://cloud.githubusercontent.com/assets/2305591/23682047/ba72f63e-0389-11e7-8add-88fb8dd21619.jpg)

**Comment by mantagen** [comment]
Can we print more of the old ones too?

Open sans, is that the font?

**Comment by Cleop** [comment]
What do people think re the F&C text? I think visually it might look cleaner without but in terms of impact (people knowing who we are) perhaps it's better to keep it?

**Comment by mantagen** [comment]
Or could change the text to read "FAC"

**Comment by iteles** [comment]
@sohilpandya It might be worth defining the aim of the t-shirts

**Comment by emilyb7** [comment]
Looking much better with the greenish colour text! I think I prefer the second of the white options, not a big fan of having branding splashed over both sides of the tshirt.

**Comment by ghost** [comment]
I think it looks nice with a small lightbulb on the front (off to the side) and the Founders & Coders on the back.

The F&C on the front as well as the lightbulb might be a bit too much? I don't know..

With respect to the text on the back I think the font family still needs to be updated to the correct font, as I don't think it matches that of the logo

And yes agreed with @iteles that the merch order should have an intended purpose and audience? Are they to give away? Are they only for F&C members? Are they to wear at events to distinguish ourselves? Are they to sell as merchandise to raise money for F&C? etc etc

**Comment by jsms90** [comment]
Loving your work @sohilpandya! :heart:

@markwilliamfirth As far as I'm concerned, getting t-shirts is about partly about fulfilling the "re-thinking coding for everyone" proposal https://github.com/foundersandcoders/london-programme/issues/149, wher it suggests that current cohort members should wear t-shirts.

It might also be nice for anyone going to meetups to network - see the `freelance-networking-and-resources` repo https://github.com/foundersandcoders/freelance-networking-and-resources/issues/1

Personally, I'm not sure it makes sense for Founders and Coders to be spending money on these just to give them away. But it was always my understanding that each cohort member received one (if they wanted it)?

Maybe other people have other ideas about what they for, as well?

**Comment by iteles** [comment]
Again, it would be good to define the objective of making these. Are they a "Here's a tshirt you might want to wear ocassionally because we're cool and we want you to think of us" or a "We wear these to meetup, talks and around town and they are therefore a vehicle for publicity"? One may warrant larger/more branding than the other. 

The purpose is certainly not just to 'look good'. 

Loving these though!

**Comment by sohilpandya** [comment]
The main reason behind the T-shirts is that we are about to run out of the old lot and we need some new ones for the students. 

@iteles can a T-shirt not be cool and worn to meetups? an all in one? I feel the dwyl one is quite nice, I literally wear that one **every day** 😟, it's not loud but people still ask me what dwyl.io stands for. 

Most importantly the new t-shirt will be a great addition to my wardrobe 😉 

@markwilliamfirth I am aware the font is still incorrect, looking to find the correct font. 👍 

So can you all please give me some reactions to the items below. 
- F&C  (Thumbs up reaction)
- FAC (Tada reaction)
- Design 1 -> image 1 & 2 (Heart reaction)

**Comment by emilyb7** [comment]
Neither F&C nor FAC mean anything to most normal people so I wouldn't do either.

The nice stickers we have say "Founders & Coders"

I like walking round with Founders & Coders on my tshirt and stuck on my laptop, because i think it sounds cool.

(nb: normal = not from FAC, we are abnormal)

**Comment by mantagen** [comment]
I too am not sure if it needs to say F&C on the front, just the logo?

**Comment by rebeccaradding** [comment]
I agree with @emilyb7 - leave the abbreviation off the shirt.

**Comment by sohilpandya** [comment]
Cool. Done. lets go with just the logo. 

Any other changes? OR I can come up with a final design for black and white Tees.

**Comment by ghost** [comment]
<img width="280" alt="screen shot 2017-03-09 at 11 22 28" src="https://cloud.githubusercontent.com/assets/5732209/23748247/bc0fe7fe-04ba-11e7-9847-3078cd9f2759.png">
<img width="289" alt="screen shot 2017-03-09 at 11 22 36" src="https://cloud.githubusercontent.com/assets/5732209/23748248/bc23740e-04ba-11e7-9883-4a662c3d3d48.png">

I would suggest something akin to this, but I think it would be advantageous to have a defined purpose, audience (effectively business proposal) for these that would therefore infer the design and quantity required. It might also be advantageous for someone to draw up some basic brand guidelines before proceeding with choosing tshirt colours and fonts etc

**Comment by iteles** [comment]
@sohilpandya I wasn't saying they can't be nice LOL
I was just saying some decisions are made easier when a purpose is clear 😊

**Comment by sohilpandya** [comment]
@markwilliamfirth I am personally not a fan of that particular text as the coders text is bigger than the founders, I would like it to be the same font size. What are your thoughts? 

Usual audience for this T-shirt consists of meetup members, Client sprint designs (e.g. Anna Frued), potential pitching at 3rd party meetups as a freelancer from Founders and Coders. 

Future audience includes teaching school kids 🎉

**Comment by ghost** [comment]
I don't think we should buy (potentially a lot of?) tshirts and spend F&C money without thinking about this first. Before we go ahead with this the following steps should take place:

Review of the Founders & Coders brand
- Review of the current logo text
- Possible creation of a new text logo
- Basic branding guidelines to be created including definitive hex codes, font families and font sizes and a basic organisation colour palette

Business proposal for tshirt order
- Why the tshirts are required and who are they to be worn by, whether they are to be given away and kept by people or lent out and returned to F&C
- Who will see them / where they will be used
- A proposed quantity based on the above needs and an estimate of costs
- A design based on the above (e.g. an embroidered lightbulb will look much better and last longer, but be more expensive if a large quantity is required)
- If there is not currently a budget that the estimate falls within or if it exceeds a reasonable amount it should be put forward at a F&C business meeting and/or approved by the finance committee. If it's an urgent requirement that can't wait until then a proposal should be written up online.

**Comment by ghost** [comment]
One might argue that if the purpose is mainly for [evening] meet ups and they are to be swapped out between people it might be a better idea to order pullover jumpers that can easily be taken off and on and swapped between people and don't need to be washed as often etc

What we're trying to do here completely depends on the purpose and audience

**Comment by emilyb7** [comment]
Jumpers would be great idea anyway, it's rarely warm enough at FAC (London) to walk around in just a tshirt!

**Comment by claireinez** [comment]
With regards to having writing on the back of a t-shirt, just remember how often you wear a jacket/cardigan in this country so the writing would rarely be seen. Is the logo on the front enough?

**Comment by sohilpandya** [comment]
OK, so should we be moving away from Tees and thinking about Jumpers/fleece/jackets? 

If we do that, then we should be coughing up the cost and not FAC. If we generate significant interest I'm happy to take a lead and get some made 🤔

**Comment by ghost** [comment]
@sofer a few questions about the F&C brand:

1. Are there currently any brand guidelines for F&C? Is there a particular font used for the logo text?
2. Is there an up-to-date Google Drive branding folder or a repo on Github for our branding assets? [This](https://github.com/foundersandcoders/assets/blob/master/faclogo.png) logo in the assets folder appears to be different to [the one on Twitter](https://pbs.twimg.com/profile_images/534123785817829376/UE8T_TQF.png). Which is the correct one?
3. Is [this](http://www.foundersandcoders.com/assets/fac-logo.png) logo the most up to date version of the lightbulb (only) logo? I'm aware it's gone through some [iterations](https://d3c5s1hmka2e2b.cloudfront.net/uploads/topic/image/185/favicon.png) over the past few years

**Comment by sohilpandya** [comment]
OK, so what are we doing with this? @sofer @markwilliamfirth

**Comment by ghost** [comment]
@sohilpandya

@nogainbar and myself are currently doing competitor branding visual research https://github.com/foundersandcoders/assets/issues/10

@nogainbar @NataliaLKB @andrewMacmurray and myself are meeting on 8th May for an initial branding ideation session for a potential new logo and new brand guidelines https://github.com/foundersandcoders/assets/issues/11

@nogainbar is also currently writing up current brand guidelines for the Founders & Coders brand https://github.com/foundersandcoders/assets/issues/9

This means that tshirts can be ordered either when issue 9 is done (deadline is before 8th may) or after new logo and brand guidelines are created (expected sometime in June)

**Comment by iteles** [comment]
I would suggest we do it in June/July. Much better to have everything aligned with new guidelines

**Comment by shiryz** [comment]
@sohilpandya @sofer @markwilliamfirth (or anyone who knows), any news on this?
we're about to order some tees for Naz (https://github.com/foundersandcoders/nazareth-programme/issues/158) and it would be helpful to know if alumni should order as well to replace the old ones.
btw I prefer a t-shirt on a jacket, we're basically melting from the heat over here :sweat_smile:

**Comment by sohilpandya** [comment]
I’m waiting for the new design before I order so in the same position as you Shireen.

**Comment by ghost** [comment]
Last update on this was @nogainbar was designing 3 new logo variants
https://github.com/foundersandcoders/assets/issues/14

But Noga said she's very busy at the moment and hasn't had time to devote to it

Current brand guidelines are [here](https://drive.google.com/file/d/0B24Sdkg-P6Mna0t3UXNGdUc4N3M/view) https://github.com/foundersandcoders/assets/issues/9 although they're not correct as the logo at the top of the page and the logo in the guideline don't match

So essentially we need to either wait until Noga has time to devote to it or get another designer on the case if this is a priority

**Comment by sofer** [comment]
I am in favour of just printing some of the existing ones for old times' sake. Our printer still has the screens. I will take 20 for starters (Male, Large).

**Comment by NataliaLKB** [comment]
@sofer seems like a good idea! That works for London, but can they make digital copies of the screens to send to @shiryz ?

Unless you can take a big suitcase with you

**Comment by ghost** [comment]
@sofer yeah the current guidelines I mentioned just have the old t-shirt design. So until new logos/designs are made then that's what it is. Do you have the design file? I thought @adamdiy might have it, but from [this thread](https://github.com/foundersandcoders/assets/issues/9#issuecomment-313671706) I'm not sure if he checks GitHub

**Comment by ghost** [comment]
@shiryz @sofer info on original designs and how to reorder is here: https://github.com/foundersandcoders/assets/issues/9#issuecomment-318638215

**Comment by yvonne-liu** [comment]
@markwilliamfirth thanks for the info. I'm taking on ordering for now. 

I've sent an email to Christopher Murphy at info (at) thepositivepress.co.uk, he hasn't responded but I was able to track down someone at the Print Club London (where he operates from) who told me Christopher is on holiday for a _few weeks_ and they're not sure if that means two weeks or four weeks. For now I have another print contact from the same location -- and I will reach out to Sophie Alda at sophiealda (at) icloud.com to see if she's able to give me the design files to start.

**Comment by yvonne-liu** [comment]
Thanks very much to @joeylouise this is our current inventory in London: 

Ladies white
Small - 6
Medium - 2
XL - 6

Men white
XL - 1

Ladies black
Small - 5
Medium - 1
XL - 1

Men black
XL - 2

@mattlub

**Comment by mattlub** [comment]
@joeylouise thanks so much- I knew there was some good stuff there and I think the students in Nazareth would definitely want them- @mantagen could you bring some out? (all the mens, ladies: all the M, probably a couple of S and XL)

**Comment by mantagen** [comment]
@joeylouise, nice
@mattlub, thanks for thinking of me pal. Was just chatting with @yvonne-liu about this yesterday. Currently the 'issue' is that I'm not looking at coming out there directly. It looks like I'll be coming there from Sardinia. However, that isn't _set in stone_. Even if I do come via Sardinia, I might be able to bring extra stuff.
@ the rest, can't wait to be wearing more of these t-shirts (white, L and XL).

**Comment by sofer** [comment]
This is the inventory and order that we made last time in 2015.

 | | Men's |  | Women's | |
-- | -- | -- | -- | --
INVENTORY | Black | White | Black | White
Small | 15 | 16 | 6 | 5
Medium | 5 | 10 | 10 | 12
Large |   | 10 | 7 | 9
X-tra Large | 6 | 9 |   | 10
  |   |   |   |  
ORDER |   |   |   |  
Small |   |   | 20 | 10
Medium | 20 |   | 10 |  
Large | 40 | 20 | 20 |  
X-tra Large | 20 |   | 20 |

**Comment by yvonne-liu** [comment]
Heard back from Chris. Our last order in 2015 cost £333 for 90 shirts. 

Following's @sofer's format:

Current inventory, as of August 2017: 

 | | Men's |  | Women's | |
-- | -- | -- | -- | --
INVENTORY | Black | White | Black | White
Small | 0 | 0 | 5 | 6
Medium | 0 | 0 | 1 | 2
Large | 0  | 0 | 0 | 0
X-tra Large | 2 | 1| 1  | 6
  |   |   |   |  
PROPOSED ORDER QTY |   |   |   |  
Small |   |   |  | 
Medium | 5? | 5?  |  |  
Large | 10 | 10 | 5? |  5?
X-tra Large |  |   |  |  

We can keep it at 20 PCS, but also happy to get a quote for 20-40PCS.

**Comment by mantagen** [comment]
I'd certainly wear an XL (and pay for it), also happy for L though if XL
confuses things.

**Comment by sofer** [comment]
@yvonne-liu I would go for a bigger order. 90 shirts will go quickly.

**Comment by yvonne-liu** [comment]
@sofer sure thing, I'll get in touch with Chris.

**Comment by yvonne-liu** [comment]
Updated order quantity to 90:

 | | Men's |  | Women's | |
-- | -- | -- | -- | --
INVENTORY | Black | White | Black | White
Small | 0 | 0 | 5 | 6
Medium | 0 | 0 | 1 | 2
Large | 0  | 0 | 0 | 0
X-tra Large | 2 | 1| 1  | 6
  |   |   |   |  
PROPOSED ORDER QTY |   |   |   |  
Small | 6 | 6  |  | 
Medium | 10 | 10  | 3 | 3 
Large | 10 | 10 | 10 |  10
X-tra Large | 4 | 5  | 3 |
